nixos/doc/manual/from_md/release-notes/rl-2111.section.xml

```diff
@@ -75,7 +75,13 @@
   </section>
   <section xml:id="other-notable-changes">
     <title>Other Notable Changes</title>
-    <para>
-    </para>
+    <itemizedlist spacing="compact">
+      <listitem>
+        <para>
+          The <literal>bitwarden_rs</literal> packages and modules were
```

Comment:
You should list explicitly that the user and service names changed and anything else that that might be relevant. If users are required to take any action you could mention here, if not you could mention that was well.

Reply:
I've added a more detailed release note, but I think it might be _too_ detailed now.

Reply:
hahaha probably - but that is much better than too little :+1: